config/memes.php

```diff
@@ -674,24 +674,65 @@
             'image' => "<img src='./images/memes/Banhammer.png' />",
             'text' => ':banhammer:',
             'public' => true,
-    ],  
+    ],
     [
             'code' => ':200iq:',
             'image' => "<img src='./images/memes/200IQ.png' />",
             'text' => ':200iq:',
             'public' => true,
-    ],  
+    ],
     [
             'code' => ':galaxybrain:',
             'image' => "<img src='./images/memes/GalaxyBrain.png' />",
             'text' => ':galaxybrain:',
             'public' => true,
-    ],  
+    ],
     [
             'code' => ':bigbrainbobby:',
             'image' => "<img src='./images/memes/BigBrainBobby.png' />",
             'text' => ':bigbrainbobby:',
             'public' => true,
-    ],      
-];
-
+    ],
+    [
+        'code' => ':elmofire:',
+        'image' => "<img src='./images/memes/elmo_fire.png' />",
+        'text' => ':elmofire:',
+        'public' => true,
+    ],
+    [
+        'code' => ':rotenkohead:',
+        'image' => "<img src='./images/memes/rotenko_head.png' />",
+        'text' => ':rotenkohead:',
+        'public' => true,
+    ],
+    [
+        'code' => ':notabit:',
+        'image' => "<img src='./images/memes/notabit.png' />",
+        'text' => ':notabit:',
+        'public' => true,
+    ],
+    [
+        'code' => ':praisethesun:',
+        'image' => "<img src='./images/memes/PraiseTheSun.png' />",
+        'text' => ':praisethesun:',
+        'public' => true,
+    ],
+    [
+        'code' => ':gahorse:',
+        'image' => "<img src='./images/memes/gahorse.gif' />",
+        'text' => ':gahorse:',
+        'public' => true,
+    ],
+    [
+        'code' => ':smugbaka:',
+        'image' => "<img src='./images/memes/smugbaka.png' />",
+        'text' => ':smugbaka:',
+        'public' => true,
+    ],
+    [
+        'code' => ':cooljojo:',
+        'image' => "<img src='./images/memes/cooljojo.gif' />",
+        'text' => ':cooljojo:',
+        'public' => true,
+    ],
+];
```

Comment:
We should actually delete this file, it doesn't matter anymore (memes are auto-generated from image files in the `memes` directory